sdk/storage/azure-storage-blob-stress/templates/stress-test-job.yaml

```diff
@@ -41,8 +41,10 @@ spec:
           {{ ternary "--sync" "" .Stress.sync }} \
           {{ ternary "--faults" "" .Stress.faults }} \
           --warmup 0 \
-          2>&1 | tee -a "${DEBUG_SHARE}/{{ .Stress.testScenario }}-`date +%s`.log";
-
+          2>&1 | tee -a "${DEBUG_SHARE}/{{ .Stress.testScenario }}-`date +%s`.log"; \
+          yum install -y procps; \
+          sleep 10; \
```

Comment:
@lmolkova the `sleep 10` was just my placeholder for an actual test command, you don't need it.

Also I would move the `procps` install into the Dockerfile if you can.

Reply:
thanks, fixed both